ErrorBlocks.md

```diff
@@ -0,0 +1,156 @@
+
+<details><p><summary>Notes</summary>
+- These volumes could also be created in another CAD program and then assigned a material in the exact same way, but this is an example of how to create these volumes in Trelis and Cubit.
+<br/></br>
+- Most likely the volumes will be slightly off centered in the x direction, but the larger bricks will be centered with one another. This may vary based on the initial CAD program.
+<br/></br>
+- To see the blocks better you can select the wireframe option in the top toolbar. It is directly to the right of the undo arrow on the toolbar. Also you can click on the axis on the bottom left of the volume viewing area to get a straight on (x,y) view.
+</p></details>
+
+#### Imprinting and Merging Volumes in Cubit/Trelis
+_Imprinting and Merging surfaces accelerates the DAGMC process by combining surfaces._
+
+  1. Imprint and Merge all shared surfaces in the model.
+     * In command panel select __Geometry -> Volume -> Imprint Merge__ and select __Imprint and Merge__ from the dropdown menu. Then type "all" into the __Volume ID(s)__ box and click __Apply__.
+     * In command line type `imprint volume all` and then enter. Then type `merge volume all`.
+     * _Now all surfaces that previously overlapped have been merged into one surface shared by both volumes._
+
+<details><p><summary>Notes</summary>
+- The default tolerated distance for imprinting/merging surfaces is 5.0e-4 units.
+<br/>
+- Combining surfaces cuts two surfaces and two sets of tetrahedron mesh areas down to one, decreasing computational power required as well as eliminating possible errors where particles exist simultaneously in two surfaces.
+</p></details>
+
+#### Assigning Materials in Cubit/Trelis
+_Most commands for this section are best implemented through the command line. More general instructions can
+be found_ [here](http://svalinn.github.io/DAGMC/usersguide/codes/dag-mcnp.html).
+
+  1. The general command for assigning Materials/Densities in Cubit/Trelis
+  `group "mat:[assigned material number]/rho:[assigned density]" add

```

Comment:
```suggestion
  `group "mat:[assigned material number or name]/rho:[assigned density]" add

```

I think whether it is a number or name is code dependent or dependent on how your materials are being handled?